label_studio/server.py

```diff
@@ -306,6 +306,15 @@ def main():
         calculate_stats_all_orgs(input_args.from_scratch, redis=True)
         return
 
+    if input_args.command == 'export':
+        from tasks.functions import export_project
+        filename = export_project(
+            input_args.project_id, input_args.export_format, input_args.path
+        )
+
+        print(f'Project exported successfully: {filename}')
```

Comment:
1 exported failed:
![image](https://user-images.githubusercontent.com/501780/185273267-73e834a3-dfa3-49a4-8eec-674a27fe33bc.png)
but "project exported successfully"

2 let's use logger instead of print, we avoid using print because it has a bad integration with datadog and other log systems